src/wasm-emscripten.h

```diff
@@ -29,6 +29,8 @@ void generateMemoryGrowthFunction(Module&);
 // signature in the indirect function table.
 std::vector<Function*> makeDynCallThunks(Module& wasm, std::vector<Name> const& tableSegmentData);
 
+void addGlobalImports(Module& wasm, std::unordered_set<Name> const& globalImports);
```

Comment:
Maybe we should call these 'object imports' instead of 'global imports' since 'object' is the counterpart to 'function', the other kind of thing that can be imported (in the linker/ELF-like terminology that we are using). Functions and objects (i.e. the symbol's type in ELF terminology) can each be global or not (i.e. the symbol's binding in ELF terminology).